spec/array_of_integer.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../part1/array_of_integer'
+require 'pry'
+
+RSpec.describe 'array_of_integer?' do
+  subject { [] }
```

Comment:
just for you to feel the difference between `subject` and `let()`
https://stackoverflow.com/questions/38437162/whats-the-difference-between-rspecs-subject-and-let-when-should-they-be-used